deploy/legionPipeline.groovy

```diff
@@ -0,0 +1,222 @@
+def installTools(){
+    sh '''
+    sudo rm -rf .venv
+    virtualenv .venv -p $(which python3)
+    cd legion_test
+    ../.venv/bin/python3 setup.py develop
+    '''
+}
+
+def buildDescription(){
+   currentBuild.description = "${params.Profile} ${params.GitBranch}"
```

Comment:
We should use different messages for each Jenkins job because they contain different parameters to print (for example build description should contain base and local version, but not profile)

Reply:
duplicate

Reply:
This PR doesn't do anything with build phase

Reply:
ok, agree